### ISSUE TITLE
500 Internal server with shows missing from Sonarr

### ISSUE THREAD

**Issue by emilnymann** [morpheus65535/bazarr#579]
**Describe the bug**
Entries that show up in history for shows that have later been removed from Sonarr will display with the name "None" and clicking it will result in a 500 internal server error with the following info:

Exception:
`TypeError("'NoneType' object has no attribute '__getitem__'",)`

Traceback:
```
Traceback (most recent call last):
  File "/home/hd27/emilnymann96/apps/bazarr/bazarr/../libs/bottle.py", line 862, in _handle
    return route.call(**args)
  File "/home/hd27/emilnymann96/apps/bazarr/bazarr/../libs/bottle.py", line 1740, in wrapper
    rv = callback(*a, **ka)
  File "/home/hd27/emilnymann96/apps/bazarr/bazarr/main.py", line 139, in wrapper
    return func(*a, **ka)
  File "/home/hd27/emilnymann96/apps/bazarr/bazarr/main.py", line 728, in episodes
    tvdbid = series_details[5]
TypeError: 'NoneType' object has no attribute '__getitem__'
```

**To Reproduce**
1. Add a show and episodes to Sonarr.
2. Wait for Bazarr to download subtitles for the episodes.
3. Delete the show in Sonarr and confirm deleting all files and show folder.
4. Go to Bazarr history
5. Click the "None" name from the now missing show.

**Expected behavior**
Handling of shows and episodes that have been deleted.

**Screenshots**
![screenshot](https://user-images.githubusercontent.com/27289912/65145396-ca06ed80-da19-11e9-8e41-8b655a3c95cc.png)

**Software (please complete the following information):**
 - Bazarr: 0.8
 - Sonarr version 3.0.3.638
 - OS: Linux Debian 18.04

**Comment by morpheus65535** [comment]
Please update to 0.8.1 and confirm if the issue is still there. One thing for sure, this won’t happen again in 0.8.2 in dev now as we won’t list missing series entries anymore.

**Comment by morpheus65535** [comment]
In fact, I won’t fix it anyway in 0.8.1. Thanks for reporting.
